homeassistant/components/reolink/host.py

```diff
@@ -277,9 +275,8 @@ async def register_webhook(self) -> None:
 
     async def unregister_webhook(self):
```

Comment:
This doesn't need to be a coroutine function since there's no await inside. Same goes for `register_webhook`.

Reply:
True, thanks for the feedback, I have converted register_webhook and unregister_webhook to normal functions.